app/Http/Controllers/SettingsController.php

```diff
@@ -363,6 +365,36 @@ public function deleteTag($tagId)
         return redirect()->route('settings.tags.index')
                 ->with('success', trans('settings.tags_list_delete_success'));
     }
+    
+    /**
+     * Method editTag
+     *
+     * @param Tag $tag 
+     * @param Request $request
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    
+    public function editTag(Tag $tag, Request $request)
```

Comment:
```suggestion
    public function editTag(Tag $tag, Request $request): RedirectResponse
```
Shall we add the return type?